tests/delete_item_test.go

```diff
@@ -0,0 +1,83 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/oolio-group/dynago"
+)
+
+type user struct {
+	Pk string `dynamodbav:"pk"`
+	Sk string `dynamodbav:"sk"`
+}
+
+func TestDeleteItem(t *testing.T) {
+	table := prepareTable(t, dynamoEndpoint, "delete_test")
+
+	ctx := context.TODO()
+
+	items := []user{
+		{Pk: "users#org1", Sk: "user#1"},
+		{Pk: "users#org1", Sk: "user#2"},
+		{Pk: "users#org1", Sk: "user#3"},
+	}
+
+	for _, item := range items {
+		err := table.PutItem(ctx, dynago.StringValue(item.Pk), dynago.StringValue(item.Sk), item)
+		if err != nil {
+			t.Fatalf("failed to insert item %v; got %s", item, err)
+		}
+	}
+
+	//query all items to check they exist
+	for _, item := range items {
+		var output []user
+		_, err := table.Query(ctx, "pk = :pk and sk = :sk", map[string]types.AttributeValue{
```

Comment:
Use GetItem or perform a single query here. It doesn't really matter much as far as output is concerned. However, it shows lack of understanding or concern about DynamoDB

Reply:
<img width="1937" alt="image" src="https://github.com/user-attachments/assets/600498d6-1357-488a-8f53-595ce5f47ad6">
Updated with GetItem for fetching single items